### PR TITLE
DOC: use the sphinx-panels extension for cards instead of raw html

### PR DESCRIPTION
**Pull request by jorisvandenbossche**
This is an alternative for https://github.com/pandas-dev/pandas/pull/32065, using the https://sphinx-panels.readthedocs.io/en/latest/ extension (which was created inspired on how we are using this in our docs, but then nicely packaged in a sphinx extension to avoid the use of raw html).

### CODE REVIEW

**Comment by jreback** [comment]
@jorisvandenbossche still relevant?

**Comment by jbrockmendel** [comment]
@jorisvandenbossche gentle ping

**Comment by jreback** [comment]
@jorisvandenbossche ?

**Comment by jreback** [comment]
@jorisvandenbossche pls update or close

**Comment by jorisvandenbossche** [comment]
@dsaxton I updated this PR to also convert the front page cards to use sphinx-panels, and with some custom styling to keep the same look and feel, I get this:

image

The "buttons" are using the default darker grey and sharper corners from sphinx-panels, but if we prefer the lighter gray / more rounded corners as it is now, that's easy to change as well. 

For reference, screenshot of the current https://pandas.pydata.org/docs/dev/:

image

**Comment by dsaxton** [comment]
> @dsaxton I updated this PR to also convert the front page cards to use sphinx-panels, and with some custom styling to keep the same look and feel, I get this:

Nice. The stronger contrast with the dark button color does make them a bit more readable. This is subjective though so I'm curious how others feel.

**Comment by jreback** [comment]
+1 on the look and not using raw html

File: doc/source/getting_started/index.rst [hunk 1]
```diff
@@ -9,82 +9,49 @@ Getting started
...
-pandas can be installed via pip from `PyPI <https://pypi.org/project/pandas>`__.
+    ++++
 
-.. raw:: html
+    .. code-block:: bash
 
-                    </p>
-                </div>
-                <div class="card-footer text-muted">
+        conda install pandas
```
Comment (tupui) [inline doc/source/getting_started/index.rst:39]: This should be
```suggestion
        pip install pandas
```

Comment (jorisvandenbossche) [reply doc/source/getting_started/index.rst]: Whoops, copy/paste error ;) Thanks for the catch!

**Pull request merged**